client/src/lib/components/sandbox/play/utils.ts

```diff
@@ -1,5 +1,6 @@
-export const makePiece = (x: number, y: number, radius: number, fill: string) => {
+export const makePiece = (gamePieceId: number, x: number, y: number, radius: number, fill: string) => {
   const piece = {
+    gamePieceId: gamePieceId,
```

Comment:
Keeping the GamePiece ID here so we can match between `DrawnPiece` and `GamePiece` instances.